docker/production/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Start with the latest Node.js LTS release
+FROM node:18-bullseye-slim
+
+# Set env variables
+ENV NODE_ENV production
+ENV APP_HOME=/opt/node/app
+
+USER node:node
+WORKDIR $APP_HOME
+
+COPY --chown=node:node package*.json .
+
+# We install Husky via a "prepare" lifecycle script - disable it in prod
+# See: https://typicode.github.io/husky/guide.html#disable-husky-in-ci-docker-prod
+RUN npm pkg delete scripts.prepare
```

Comment:
`--ignore-scripts` also ignores other lifecycle scripts like prepublish etc. We don't use them now, but I didn't want this to be a surprise later in case we do use it.

I think layers are inexpensive in Docker. If there is no change to package.json, then even this layer will be cached on subsequent runs.